cli/cli-client/src/main/resources/operator/rbac.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
@tristantarrant please pay attention, that Role now is under rights assessment
https://github.com/infinispan/infinispan-operator/pull/537